float64 width # [m]
float64 velocity # [m/s]
